dist/XSLoader/t/XSLoader.t

```diff
@@ -117,6 +117,7 @@ SKIP: {
 
```

Comment:
easier to get in upstream by adding a `use warnings` at the top?

Reply:
Noted, see commit messages in updated p.r.